services_app/src/test/java/org/opendatakit/services/sync/actions/viewModels/viewModels/AbsSyncViewModelTest.java

```diff
@@ -111,4 +120,117 @@ public void checkIfSyncAttachmentState_isAvailable() {
         assertEquals(SyncAttachmentState.REDUCED_SYNC, absSyncViewModel.getCurrentSyncAttachmentState());
         assertNotNull(absSyncViewModel.getSyncAttachmentState().getValue());
     }
+
+    @Test
+    public void testNullAppName() {
+        absSyncViewModel.setAppName(null);
+        assertNull(absSyncViewModel.getAppName());
+    }
+
+    @Test
+    public void testNullServerUrl() {
```

Comment:
I think a single test checking null behavior for all three(`testNullAppName`, `testNullServerUrl`,`testNullUsername`) could suffice. Since  testing each one individually might be redundant. @wbrunette what are your thoughts, please?

Reply:
I'll combine these into one as well.